modules/administration-guide/pages/configuring-storage.adoc

```diff
@@ -7,4 +7,9 @@
 [id="configuring-storage"]
 = Configuring storage
 
+[WARNING]
+====
+Network File Storage (NFS) is not supported.
```

Comment:
Looks like it's [System](https://duckduckgo.com/?q=kubernetes+network+file+system+nfs&t=ftsa&atb=v320-1&ia=web), not `Storage` (regardless of what somebody might write in their blog on the Internet):
```
{prod-short} does not support Network File System (NFS).
```